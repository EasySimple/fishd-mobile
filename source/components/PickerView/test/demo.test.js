import webDemoTest from '../../../tests/demoTest';

webDemoTest('PickerView');
