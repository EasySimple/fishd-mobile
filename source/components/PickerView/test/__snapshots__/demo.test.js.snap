// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders PickerView demo correctly 1`] = `
<div>
  <div
    class="components-tpl-demo-basic"
  >
    <div
      class="demo-title"
    >
      基础用法
    </div>
    <div
      class="demo-card"
    >
      <div
        class="fm-picker"
        style="flex-direction:row"
      >
        <div
          class="fm-picker-col"
          style="flex:1"
        >
          <div
            class="fm-picker-col-mask"
          />
          <div
            class="fm-picker-col-indicator "
          />
          <div
            class="fm-picker-col-content"
          >
            <div
              class="fm-picker-col-item fm-picker-col-item-selected "
            >
              2013
            </div>
            <div
              class="fm-picker-col-item "
            >
              2014
            </div>
          </div>
        </div>
        <div
          class="fm-picker-col"
          style="flex:1"
        >
          <div
            class="fm-picker-col-mask"
          />
          <div
            class="fm-picker-col-indicator "
          />
          <div
            class="fm-picker-col-content"
          >
            <div
              class="fm-picker-col-item fm-picker-col-item-selected "
            >
              春
            </div>
            <div
              class="fm-picker-col-item "
            >
              夏
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    class="components-tpl-demo-basic"
  >
    <div
      class="demo-title"
    >
      单选
    </div>
    <div
      class="demo-card"
    >
      <div
        class="fm-picker"
        style="flex-direction:row"
      >
        <div
          class="fm-picker-col"
          style="flex:1"
        >
          <div
            class="fm-picker-col-mask"
          />
          <div
            class="fm-picker-col-indicator "
          />
          <div
            class="fm-picker-col-content"
          >
            <div
              class="fm-picker-col-item fm-picker-col-item-selected "
            >
              春
            </div>
            <div
              class="fm-picker-col-item "
            >
              夏
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    class="components-tpl-demo-basic"
  >
    <div
      class="demo-title"
    >
      级联
    </div>
    <div
      class="demo-card"
    >
      <div
        class="fm-picker"
        style="flex-direction:row;align-items:center"
      >
        <div
          class="fm-picker-col"
          style="flex:1"
        >
          <div
            class="fm-picker-col-mask"
          />
          <div
            class="fm-picker-col-indicator "
          />
          <div
            class="fm-picker-col-content"
          >
            <div
              class="fm-picker-col-item fm-picker-col-item-selected "
            >
              北京
            </div>
            <div
              class="fm-picker-col-item "
            >
              浙江
            </div>
          </div>
        </div>
        <div
          class="fm-picker-col"
          style="flex:1"
        >
          <div
            class="fm-picker-col-mask"
          />
          <div
            class="fm-picker-col-indicator "
          />
          <div
            class="fm-picker-col-content"
          >
            <div
              class="fm-picker-col-item fm-picker-col-item-selected "
            >
              东城区
            </div>
            <div
              class="fm-picker-col-item "
            >
              西城区
            </div>
            <div
              class="fm-picker-col-item "
            >
              崇文区
            </div>
            <div
              class="fm-picker-col-item "
            >
              宣武区
            </div>
          </div>
        </div>
        <div
          class="fm-picker-col"
          style="flex:1"
        >
          <div
            class="fm-picker-col-mask"
          />
          <div
            class="fm-picker-col-indicator "
          />
          <div
            class="fm-picker-col-content"
          />
        </div>
      </div>
    </div>
  </div>
</div>
`;
